torch/nn/intrinsic/qat/modules/conv_fused.py

```diff
@@ -94,7 +94,11 @@ def _forward(self, input):
         bias_shape[1] = -1
         scaled_weight = self.weight_fake_quant(self.weight * scale_factor.reshape(weight_shape))
         # this does not include the conv bias
-        conv = self._conv_forward(input, scaled_weight)
+        if self.bias:
+            zero_bias = torch.zeros_like(self.bias)
+        else:
+            zero_bias = torch.zeros(self.out_channels, device=scaled_weight.device())
```

Comment:
nice catch.  Can we add a comment here with the motivation, similar to this PR summary?

Reply:
sure